Dockerfile

```diff
@@ -25,7 +25,7 @@ ARG IPFS_PLUGINS
 # Also: fix getting HEAD commit hash via git rev-parse.
 RUN cd $SRC_DIR \
   && mkdir -p .git/objects \
-  && make build GOTAGS=openssl IPFS_PLUGINS=$IPFS_PLUGINS
+  && GOFLAGS=-buildvcs=false make build GOTAGS=openssl IPFS_PLUGINS=$IPFS_PLUGINS
```

Comment:
This is needed because Go 1.18 introduced add version control information into binaries whenever it thinks VCS is present.

Because we have a pretty bare git repo for the purposes of computing the HEAD and putting that information into the binary commands like `git status` won't work which will cause a build failure.

GOFLAGS is being prefixed instead of suffixed here because we also need it set for commands like `go list` which apparently require VCS as well now.

If we're fine with this for the time being we can writeup a new issue for deciding if/how we want to leverage VCS information inside of docker builds in the future.

Reply:
good to me